python-minimal-streamlit/environment.yml

```diff
@@ -0,0 +1,16 @@
+name: "base"
+channels:
+  - defaults
+  - conda-forge
+dependencies:
+  - numpy
+  - pandas
+  - scipy
+  - matplotlib
+  - seaborn
+  - plotnine
+  - pip:
+    - sklearn
```

Comment:
these are probably not all needed to run streamlit?

Reply:
No, they are not. These are the packages I'd like to start a project with. 
Do you want this to be a streamlit only template or can it be a bit more?

Reply:
I'd say in this repo we should keep these templates as lean as possible.